components/crypto/README.md

```diff
@@ -0,0 +1,75 @@
+# Cryptography primitives for Exonum
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/blob/master/LICENSE)
+![rust 1.36.0+ required](https://img.shields.io/badge/rust-1.36.0+-blue.svg?label=Required%20Rust)
+
+`exonum-crypto` provides a high-level API for work with various cryptography tasks.
+
+Capabilities of `exonum-crypto` include:
+
+- Calculating the hash of data;
+- Generating key pairs for work with digital signatures;
+- Creating and verifying of digital signatures.
+
+The main backend for `exonum-crypto` is `sodiumoxide`, and used algorithms are:
+
+- `SHA256` for hashing.
+- `Ed25519` for digital signatures.
+
+Consult [the crate docs](https://docs.rs/exonum-crypto) for more details.
+
+## Examples
+
+Signing data and verifying the signature:
+
+```rust
+fn main() {
+  exonum_crypto::init();
```

Comment:
Nit: Why did you use 2-space indentation here, but 4-space elsewhere? 🙃

Reply:
Sublime Text for some reason decided that tab there should equal to two spaces ^^"